common/plugins/org.jboss.tools.common.core/.classpath

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry exported="true" kind="lib" path="lib/jandex-2.0.4.Final.jar"/>
-	<classpathentry kind="output" path="target/classes"/>
```

Comment:
Not sure this is not a local change ?

Reply:
yes, looks like so, but does not influence the way the build is performed, as tycho does not use this  info but MANIFEST and build.properties.